cmd/version_test.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVersionRootCommand(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	configYaml, err := ioutil.ReadFile("./testdata/config.yaml")
+	err = afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
+	// Locally we are not support version, that's why we see err
+	expectedOut := "Error: Astronomer CLI version is not valid\n"
```

Comment:
Shouldn't this message be on stderr, not stdout?

Reply:
Random thought - rather than passing `client *houston.Client, out io.Writer, error io.Writer` (as it would have to become to have error messages captured) to all the functions should we instead use [context](https://golang.org/pkg/context/) (I'm not too familiar with using contexts - i know it is "normally" used for http request serving for instance, so I'm not sure if it makes sense here)

Reply:
@ashb thanks for review!
IMO context here is a little bit overkill, it's for another set of problems. I checked helm/docker/k8s/doctl and this approach has been inspired by common usage. I prefer 1 step in time, instead of global vars and inits, now we have aka cmd factory to build commands and can test and control client/output. Yes some parts of old code still looks strange, but it's point to improve in future, create extra struct type to hide this details:
```
type SomeDetails struct {
    client Client
    out os.Writer
    ...
}
someD := NewSomeDetails()
// to reduce number of args
newWorkspaceCmd(someD) 
```